Userland/Services/NotificationServer/NotificationWindow.cpp

```diff
@@ -69,6 +70,11 @@ NotificationWindow::NotificationWindow(i32 client_id, String const& text, String
     set_rect(rect);
 
     auto widget = NotificationServer::NotificationWidget::try_create().release_value_but_fixme_should_propagate_errors();
+    widget->set_greedy_for_hits(true);
```

Comment:
The close button still closes the notification, right?

Reply:
Yep, I think that's part of the window (rather than the main widget).